src/ClientData/include/ClientData/DataManager.h

```diff
@@ -70,6 +70,11 @@ class DataManager final {
   [[nodiscard]] bool IsFrameTrackEnabled(const FunctionInfo& function) const;
   void ClearUserDefinedCaptureData();
 
+  void SetTimeRangeSelection(uint64_t start, uint64_t end);
+  void ClearTimeRangeSelection();
+  [[nodiscard]] uint64_t GetTimeRangeSelectionStart() const;
+  [[nodiscard]] uint64_t GetTimeRangeSelectionEnd() const;
```

Comment:
Maybe add a struct like `TimeRange` ? 
1. We would have one method instead of two here
2. Where do we check `start <= end`? With `TimeRangeSelection` you would be naturally checking this upon instantiation. 
3. This way we would reduce the chance of smth in the spirit of `SetTimeRangeSelection(end, start)`.
4. Is it inclusive or exclusive? That's neither clear from the declaration, nor from the callsite. We could name the struct like `InclusiveTimeRange` and make it clear.

Reply:
Thanks for the idea! I think it actually makes my next PR cleaner as well.  I made a comment on the struct about it being inclusive, but if you think we should still add it into the name, let me know.

Reply:
IMO, the comment is enough.

The only reason I can think of to favour `InclusiveTimeRange` is its coexistence with `ExclusiveTimeRange`. And you might want that. 

In https://github.com/google/orbit/pull/4622/files you add a `bool exclusive` argument to `TimerData::GetTimers`. I think, it would be much cleaner to have two overloads `TimerData::GetTimers(InclusiveTimeRange)` and `TimerData::GetTimers(ExclusiveTimeRange)`.

Anyway, this is out of scope of the PR.